### PR TITLE
updated nuget packages

### PR DESCRIPTION
**Pull request by bannerflow-hassaan**
Updated some nuget packages like JSON.NET

### CODE REVIEW

File: MusicMashup/MusicMashup.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.1\build\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props" Condition="Exists('..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.1\build\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props')" />
-  <Import Project="..\packages\Microsoft.Net.Compilers.1.1.1\build\Microsoft.Net.Compilers.props" Condition="Exists('..\packages\Microsoft.Net.Compilers.1.1.1\build\Microsoft.Net.Compilers.props')" />
+  <Import Project="..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.2\build\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props" Condition="Exists('..\packages\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.1.0.2\build\Microsoft.CodeDom.Providers.DotNetCompilerPlatform.props')" />
+  <Import Project="..\packages\Microsoft.Net.Compilers.1.3.2\build\Microsoft.Net.Compilers.props" Condition="Exists('..\packages\Microsoft.Net.Compilers.1.3.2\build\Microsoft.Net.Compilers.props')" />
```
Comment (syedhassaanahmed) [inline MusicMashup/MusicMashup.csproj:4]: Hope this won't be a breaking change.

**Review by syedhassaanahmed — CHANGES_REQUESTED** [review]
Please see comments in csproj

File: MusicMashup/MusicMashup.csproj (hunk 1 above)
Comment (bannerflow-hassaan) [reply MusicMashup/MusicMashup.csproj:4]: No it has been tested

**Review by bannerflow-hassaan — COMMENTED** [review]
Please check

**Review by syedhassaanahmed — APPROVED** [review]
Looks ok!

**Comment by syedhassaanahmed** [comment]
Thanks!

**Pull request merged**